app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -420,7 +424,7 @@ def ca_cert
     private
 
     def update_available_operating_systems
-      return false if errors[:url].any?
+      return false if errors.any?
```

Comment:
@ShimShtein I thought about this and maybe we should prevent loading the operating system for any error and not just errors[:base] that I can put in `connect`. If you don't agree I'll go back to checking url and base errors and change the `connect` method to set it.

Reply:
It makes sense. I think there is no point making extra oVirt API calls, if the model is not valid. I think you could just call `#valid?` here, by the way.

Reply:
I don't think there is such a difference since `valid?` also depends on the errors and if we did not update errors I'm fine with breaking because the validation is wrong.

Reply:
:ok_hand: then. I can live with that :)